cfgov/cfgov/tests/test_urls.py

```diff
@@ -0,0 +1,71 @@
+try:
```

Comment:
This is for Django 1.10+ compatibility. `RegexURLPattern` and `RegexURLResolver` moved to `django.urls` in 1.10.